src/ompl/tools/thunder/Thunder.h

```diff
@@ -97,6 +97,9 @@ namespace ompl
         private:
             /** \brief Shared constructor functions */
             void initialize();
+            double stretch_factor_ {1.2};
+            double DenseD_ {0.001};
+            double SparseD_ {0.1};
```

Comment:
where are these used?

Reply:
Also not a terrible idea to add some comments explaining what these are.